person.rb

```diff
@@ -13,6 +13,10 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  def validate_name
+    @name = @corrector.correct_name(@name)
```

Comment:
- [ ] I would suggest to make an `instance of Corrector set` first in the Person class on initialization.